pyproject.toml

```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "5.0"
+version = "5.0.1"
```

Comment:
This adds a whole new feature, right, so it would be a minor version bump, 5.1.0?

Reply:
5.1.0 it is